src/BenchmarkDotNet/Engines/Engine.cs

```diff
@@ -41,6 +41,7 @@ public class Engine : IEngine
         private bool EvaluateOverhead { get; }
         private bool MemoryRandomization { get; }
 
+        private readonly List<Measurement> jittingMeasurements = new (10);
```

Comment:
Is my understanding correct that we won't ever have more than 4 jitting measurements?

- overhead jitting, unroll factor 16
- overhead jitting, unroll factor 1
- workload jitting, unroll factor 16
- workload jitting, unroll factor 1

```suggestion
        private readonly List<Measurement> jittingMeasurements = new ();
```